core/chaincode/platforms/golang/platform.go

```diff
@@ -272,7 +272,7 @@ func DescribeCode(path string) (*CodeDescriptor, error) {
 
```

Comment:
Above, on line 166, is where I'd expect name munging to occur. The 'name' element is what is used in the name and the file path is used to stat the data.